tests/meta-data-from-ast-test.js

```diff
@@ -0,0 +1,186 @@
+'use strict';
+
+const expect = require('chai').expect;
+const metaDataFromAST = require('../lib/meta-data-from-ast');
+const defaultParser = require('../lib/default-parser');
+
+function toAST(source) {
+  return defaultParser(source);
+}
+
+describe('.metaDataFromAST', function() {
+  describe('ES5', function() {
+    const FOO = toAST(`define('foo', ['a', 'b/c'], function() { });`);
+    const A = toAST(`define('a', ['exports', 'require'], function() { })`);
+    const C = toAST(`define('c', ['../a', '../d'], function() { })`);
+    const D = toAST(`define('d', ['foo'], function() { })`);
+
+    it.only('extracts deps', function() {
+      expect(metaDataFromAST(FOO)).to.eql({
+        name: 'foo',
+        imports: {
+          a: ['default'],
+          'b/c': ['default'],
+        },
+        exports: {},
+      });
+
+      expect(metaDataFromAST(A)).to.eql({
+        name: 'a',
+        imports: {
+          exports: ['default'],
```

Comment:
Hmm, I think in the AMD as source world this is a bit harder to manage.  If we search for which keys on the import object are used (which seems fine to me) then I don't think we expect `default` here (because it doens't use `export` at all)...

Reply:
maybe `export: true` ?